docs/platforms/react-native/data-management/data-collected.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. This page lists data categories that the Sentry React Native SDK collects.
+
+Many of the categories listed here require you to set <PlatformLink to="/configuration/options/#send-default-pii">`sendDefaultPii: true`</PlatformLink> in your `Sentry.init({})` config.
+
+## HTTP Headers
+
+By default, the Sentry SDK attaches HTTP request information, such as URL, user-agent, referrer, and other headers, to the event. To disable this behavior, remove the `Sentry.httpContextIntegration` from the integrations array.
```

Comment:
`m`: do we have a denylist in code we can link to, similar to e.g. Python [here](https://docs.sentry.io/platforms/python/data-management/data-collected/#http-headers)

Reply:
The `httpContextIntegration` is quite simple, it doesn't have any denylists.

But it also doesn't collect any headers, the description in JS is outdated.

I have updated the text in https://github.com/getsentry/sentry-docs/pull/12811/commits/d3a7c62212ac26eb68fc1490827b7545290bc838.